bootstraptest/test_yjit_new_backend.rb

```diff
@@ -144,6 +144,14 @@ def foo()
     foo()
 }
 
+# opt_send_without_block (VM_METHOD_TYPE_CFUNC)
+assert_equal 'nil', %q{
+    def foo
+      nil.inspect
+    end
+    foo
+}
+
```

Comment:
Can you also add a test with a cfunc call that takes 1 argument and one that takes 2 arguments (so that argument ordering matters) ?

Reply:
Yeah, I should have. Added `argc: 1`, `argc: 2`, and `argc: -1` cases here https://github.com/Shopify/ruby/pull/357/commits/d7832356fac6e36e807abaad2b213df83f7ff419. I also added a kwargs case and fixed a bug in that 😅 https://github.com/Shopify/ruby/pull/357/commits/b655156079194030efec08a9068dd93e442ad099